recipes/matplotlib-label-lines/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "matplotlib-label-lines" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: "https://github.com/cphyc/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz"
+  sha256: 4c5ce085f7c1f99934d992024416dd3aad2cefd6bd8ea1e8837251cb19f0d9f2
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+
+  run:
+    - python >=3.6
+    - {{ pin_compatible('numpy') }}
```

You only need the `pin_compatible` if you're building against Numpy's C-API:
```suggestion
    - numpy
```